config/config.go

```diff
@@ -156,6 +156,11 @@ var (
 		RefreshInterval: model.Duration(5 * time.Minute),
 	}
 
+	// DefaultECSSDConfig is the default ECS SD configuration.
+	DefaultECSSDConfig = ECSSDConfig{
+		RefreshInterval: model.Duration(60 * time.Second),
```

Comment:
Don't get me wrong this is a great PR and of most SD mechanisms this is one that I would like to see natively, I just want to express my opinion regarding SD mechanism fragmentation in general. Also considering this SD implementation is actually tested.